src/main/resources/lang/EN.yml

```diff
@@ -51,10 +51,14 @@ del-deleted: "Warp '%0%' succesfully deleted."
 del-deletedOther: "Warp '%0%' succesfully deleted for %1%."
 
 list-header: "Warps from %0%:"
-list-public: "&7Public:&r %0%."
+list-public:
+  - "[gray]Public:[reset] %0%."
+  - "    command: /warp to %0%"
```

Comment:
It would be nice to also add a `hover:` part to indicate what will happen when you click it, examples: https://github.com/NLthijs48/AreaShop/blob/master/AreaShop/src/main/resources/lang/EN.yml

You could simply copy the `action` and `command` messages and use those to format commands like these.